firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -179,6 +182,76 @@ public interface FirebaseAppDistribution {
    */
   void startFeedback(@NonNull CharSequence infoText, @Nullable Uri screenshot);
 
+  /**
+   * Displays a notification that, when tapped, will take a screenshot of the current activity, then
+   * start a new activity to collect and submit feedback from the tester along with the screenshot.
+   *
+   * <p>On Android 13 and above, this method requires the <a
+   * href="https://developer.android.com/develop/ui/views/notifications/notification-permission">runtime
```

Comment:
Does a `*` in the middle of the HTML tag work for JavaDoc?

Reply:
According to the rendered view in Android Studio it works 👍